dev/ctests/cases/C48_S2SW_gfs_fcst_seg0.yaml

```diff
@@ -7,44 +7,541 @@
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f107.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f107.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f108.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f108.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f109.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f109.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f110.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f110.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f111.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f111.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f1{{ cyc }}.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f1{{ cyc }}.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f113.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f113.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f114.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f114.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f115.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f115.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f116.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f116.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f117.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f117.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f118.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f118.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f119.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f119.bin]
+        - [{{ SRC_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f{{ cyc }}0.bin, {{ DST_DIR }}/gfs.{{ PDY }}/{{ cyc }}/model/wave/history/gfs.wave.t{{ cyc }}z.uglo_100km.f{{ cyc }}0.bin]
```

Comment:
may be reduce to first and last.  It is unlikely if the last matches, anything in the middle won't match.

Reply:
This was supposed to be for an exhaustive list for the default validation behavior of file existence to promote the rise of a data driven workflows through this empirical exercise of fully populating these case files which intern will fail if not complete.

Reply:
discussed offline.